src/psyclone/psyir/nodes/structure_reference.py

```diff
@@ -210,6 +214,22 @@ def __str__(self):
             result += str(entity) + "\n"
         return result
 
+    @property
+    def member(self):
+        '''
+        :returns: the member of the structure that this reference is to.
+        :rtype: :py:class:`psyclone.psyir.nodes.Member`
+
+        :raises InternalError: if the first child of this node is not an \
+                               instance of Member.
+        '''
+        if not isinstance(self.children[0], Member):
+            raise InternalError(
+                "StructureReference malformed or incomplete. The first child "
```

Comment:
Since this can be inherited by AoSReference, can you replace "StructureReference" to `type(self).__name__` or something similar.

Reply:
Oops, done.